example.js

```diff
@@ -46,6 +47,14 @@ if (useCustomIcons) {
   MyModeIcon = CustomIcons.CustomModeIcon
 }
 
+// define some application-wide components that should be used in
+// various places
+const components = {
+  ItineraryBody: DefaultItinerary,
+  LegIcon: MyLegIcon,
+  ModeIcon: MyModeIcon
+}
+
```

Comment:
I was also thinking that, but it'd be a much larger change to instead include that in the context instead of the redux store. Also, perhaps some parts of the config might be changeable, for example if we ever implement a webapp with multiple languages, maybe it could have a config setting of the language that could change in the app.

Reply:
The more I think about it tho, maybe it would be good to consider the config being a global context thing that is assumed to not change. It could be useful to even explore adding some kind of context provider to otp-ui packages which would reduce the amount of prop-drilling and simplify the inclusion of config items throughout those components.